drivers/overlay/ov_network.go

```diff
@@ -291,7 +377,19 @@ func (n *network) KeyPrefix() []string {
 }
 
 func (n *network) Value() []byte {
-	b, err := json.Marshal(n.vxlanID())
+	overlayNetmap := make(map[string]interface{})
+
+	s := n.subnets[0]
```

Comment:
Are we only taking care of single subnet case for now? That is fine. Just wanted to confirm